packages/block-library/src/shortcode/editor.scss

```diff
@@ -16,9 +16,6 @@
 	}
 
 	.block-editor-plain-text {
-		flex-grow: 1;
-		line-height: 1;
-		max-height: 30px;
```

Comment:
I like the improvement being made. But I wonder If we should keep some max height value (although big) just in case a shortcode contains a really big amount of text:
![image](https://user-images.githubusercontent.com/11271197/57033300-75b6c900-6c44-11e9-9efa-1baad176588d.png)

Reply:
I thought about that and didn't mind it expanding the full height. I'm not too familiar with shortcodes myself, so willing to be overruled here. Do you feel okay merging it as is, @jorgefilipecosta ?

Reply:
I'll take another look at this because I really like how GitHub puts a height limit on their snippets of code. 

![set-height](https://user-images.githubusercontent.com/617986/57803664-cd0d6c80-770d-11e9-99b0-746b71fd5078.gif)